lambdas/db-migration/src/migrations/20201014105016_create_providers_table.ts

```diff
@@ -20,13 +20,13 @@ export const up = async (knex: Knex): Promise<void> =>
       .notNullable();
     table
       .integer('port')
-      .comment('Port name for acessing the provider');
+      .comment('Port name for accessing the provider');
     table
       .text('username')
-      .comment('Username for acessing the provider');
+      .comment('Username for accessing the provider');
     table
       .text('password')
-      .comment('password for acessing the provider');
+      .comment('password for accessing the provider');
```

Comment:
This is a nit but could we all capitalize the `p` in password? 
```suggestion
      .comment('Password for accessing the provider');
```